.gitignore

```diff
@@ -2,7 +2,7 @@
 *.swp
 dist/*
 .vscode/*
-gitsign
-gitsign-credential-cache
+/gitsign
```

Comment:
I'm wondering if this is intentional, 🤔 .

Reply:
Yup, it is! Before this would match any path that had gitsign in it, so it was ignoring the `internal/gitsign` folder. This changes it to only ignore files named gitsign in the git root dir - the primary purpose for this was to ignore gitsign binaries built via `go build .`.